thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -597,12 +597,10 @@ export default Component.extend({
       analysisRange,
       selectedBaseline
     } = this.getProperties('metricUrn', 'analysisRange', 'selectedBaseline');
-    const granularity = '15_MINUTES';
-    const timezone = moment.tz.guess();
 
     set(this, 'errorTimeseries', null);
 
-    const urlCurrent = `/rootcause/metric/timeseries?urn=${metricUrn}&start=${analysisRange[0]}&end=${analysisRange[1]}&offset=current&granularity=${granularity}&timezone=${timezone}`;
+    const urlCurrent = `/rootcause/metric/timeseries?urn=${metricUrn}&start=${analysisRange[0]}&end=${analysisRange[1]}&offset=current`;
```

Comment:
Should get the granularity and timezone from detect function.
E.g., the underline data may have hourly granularity, but we may want to monitor it in daily granularity.

Reply:
The time zone should be the frontend's timezone. 

Technically, the granularity of the graph should be the granularity for detection. However, with yaml, we allow the user to monitor in different granularity in different detection rules. Therefore, multiple granularities could be used in the in one detection, we don't know which one to show. I suggest using default time granularity for the metric in preview graph for now.